apps/framework-cli/src/cli/local_webserver.rs

```diff
@@ -3292,41 +3294,46 @@ async fn get_admin_reconciled_inframap(
             ))
         })?;
 
-    // Load current map from state storage (these are the tables under Moose management)
-    let current_map = match state_storage.load_infrastructure_map().await {
-        Ok(Some(infra_map)) => infra_map,
-        Ok(None) => InfrastructureMap::empty_from_project(project),
-        Err(e) => {
-            return Err(crate::framework::core::plan::PlanningError::Other(
-                anyhow::anyhow!("Failed to load infrastructure map from state storage: {e}"),
-            ));
-        }
-    };
-
     if !project.features.olap {
+        // If OLAP is disabled, we can't reconcile, just return stored state
```

Comment:
That's not right - the infraamp also has kafka etc... and if it is not already part of reconciliation it should eventually be part of it - we should trigger it even though it would not do anything